demo/src/index.js

```diff
@@ -18,6 +18,7 @@ class Demo extends Component {
     this.state = {
       messageList: messageHistory,
       newMessagesCount: 0,
+      typing: false,
```

Comment:
the attribute of state here should be called `isTyping` in keeping with the convention used for `isOpen`

Reply:
Ok